examples/wearable/UIComponents/contents/presentation-views/list/basic-list.html

```diff
@@ -18,7 +18,7 @@
 	<div class="ui-page">
 		<header class="ui-header">
 			<h2 class="ui-title">
-				Basic List
+				Title
```

Comment:
@korneliakobiela  and @lslachciak 
I'd like to ask whch guideline we're following (html or pdf)? According to html we should have the following structure:
 - List
 - List: Main + Sub text
 - List: Main + Sub text(editable) 
 - List: Main + Sub text text_max

Reply:
In the HTML guide on `#artboard39`, you can see a list like this